torchvision/prototype/transforms/functional/_meta.py

```diff
@@ -55,6 +55,10 @@ def get_spatial_size_image_pil(image: PIL.Image.Image) -> List[int]:
     return [height, width]
 
 
+# TODO: Should we have get_spatial_size_video here? How about masks/bbox etc? What is the criterion for deciding when
```

Comment:
`get_spatial_size` should apply to everything, right? That was the whole reason we have extracted it out, because bounding boxes and masks can provide this information, while `num_channels` is reserved for images and videos.

Reply:
I've already updated `get_spatial_size` to handle all inputs. 

I think you are trying to answer a different question from what I ask here. What I think we should discuss is whether there should be specific kernels for each type, unrelated to whether the dispatcher can handle everything. We already have kernels (like `erase_video`) that aren't necessarily used in the dispatcher. So here I'm asking, what should the convention over providing kernels for individual types should be.

Reply:
Ah, sorry, yes I was confused. That is a good question and I don't have an answer for it yet. My gut says that we should stay consistent and provide the kernels just as we do for the other transforms.

Reply:
Same feeling here. I'll leave the TODO for the follow up. I think we can answer this on the PR where we switch `image_size` to `spatial_size`